content/en/real_user_monitoring/guide/monitor-capacitor-applications-using-browser-sdk.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Monitor Capacitor Applications Using the Browser SDK
+description: Learn how to monitor cross-platform applications built with Capacitor through the browser RUM SDK.
+further_reading:
+  - link: '/real_user_monitoring/'
+    tag: 'Documentation'
+    text: 'Learn about Real User Monitoring'
+---
+
+## Overview
+
+[Capacitor][1] is an open source native runtime for building Web Native applications that run natively on iOS, Android, and progressive web applications with JavaScript, HTML, and CSS.
```

Comment:
I think there is too much `native` words here which make it confusing a bit. IS that a default capacitor definition from their website?

Reply:
Hi @evorotnikov-dd on their [website](https://capacitorjs.com/) they describe themselves like so:

A cross-platform native runtime for web apps.
Capacitor is an open source native runtime for building Web Native apps. Create cross-platform iOS, Android, and Progressive Web Apps with JavaScript, HTML, and CSS.

Would this be better?

```suggestion
[Capacitor][1] is an open source native runtime for building Web Native applications that run on iOS, Android, and progressive web applications with JavaScript, HTML, and CSS.
```